source/swap/docsTemplate/contract.hbs

```diff
@@ -34,7 +34,9 @@
 | ----------------- | :------------    |
 {{/if}}|`{{param}}`|{{description~}}{{#raw}}|{{/raw}}{{/each}}{{/if}}
 
+
 {{#if natspec.returns}}
+## Returns
```

Comment:
I think you only added the `## returns` line to one of the template docs. Please can you add it to all of them?